<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1093</width>
    <height>791</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Scene" name="sceneWidget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>19</y>
      <width>811</width>
      <height>661</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOpenFile">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>501</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
color: black;
	background-color: rgb(255, 255, 171);
border-radius: 20px;
  border: 1px solid black;
}

QPushButton:pressed {
border: 2px solid black;
            background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #CA6C00, stop: 1 #F58C14);
}

</string>
    </property>
    <property name="text">
     <string>Open file</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelLineSetting">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>30</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>LINE SETTING</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelVertexSetting">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>210</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>VERTEX SETTING</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelLineColor">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>60</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>color</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cumBoxLineColor">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>60</y>
      <width>103</width>
      <height>20</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>White</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Black</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Red</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Green</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Blue</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="labelLineThick">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>170</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>thick</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelVertexColor">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>250</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>color</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelVertexThick">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>350</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>thick</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cumBoxVertexColor">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>250</y>
      <width>119</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <item>
     <property name="text">
      <string>White</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Black</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Red</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Green</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Blue</string>
     </property>
    </item>
   </widget>
   <widget class="QSlider" name="sliderLineThick">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>170</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>40</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="sliderVertexThick">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>350</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>40</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBackground">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>420</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
color: black;
	background-color: rgb(255, 255, 171);
border-radius: 20px;
  border: 1px solid black;
}

QPushButton:pressed {
border: 2px solid black;
            background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #CA6C00, stop: 1 #F58C14);
}
</string>
    </property>
    <property name="text">
     <string>BACKGROUND SETTING</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSBScale">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>719</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="minimum">
     <double>0.100000000000000</double>
    </property>
    <property name="maximum">
     <double>2.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="btnScale">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>710</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
color: black;
	background-color: rgb(255, 255, 171);
border-radius: 15px;
  border: 1px solid black;
}

QPushButton:pressed {
border: 2px solid black;
            background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #CA6C00, stop: 1 #F58C14);
}

</string>
    </property>
    <property name="text">
     <string>ZOOM</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>550</y>
      <width>241</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayoutFileInfo">
     <property name="labelAlignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="verticalSpacing">
      <number>6</number>
     </property>
     <item row="3" column="0">
      <widget class="QLabel" name="labelFileName">
       <property name="text">
        <string>Имя файла:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="labelFileName_2">
       <property name="text">
        <string>нет файла</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <spacer name="verticalSpacer_5">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="labelCountOfVertex">
       <property name="text">
        <string>Кол-во точек:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="labelCountOfVertex_2">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <spacer name="verticalSpacer_6">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="labelCountOfFacets">
       <property name="text">
        <string>Кол-во связей:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLabel" name="labelCountOfFacets_2">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <spacer name="verticalSpacer_8">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="0">
      <spacer name="verticalSpacer_7">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btnRotate">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>710</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
color: black;
	background-color: rgb(255, 255, 171);
border-radius: 15px;
  border: 1px solid black;
}

QPushButton:pressed {
border: 2px solid black;
            background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #CA6C00, stop: 1 #F58C14);
}

</string>
    </property>
    <property name="text">
     <string>ROTATE</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>120</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_line">
     <item row="0" column="0">
      <widget class="QRadioButton" name="radioBtnLineSolid">
       <property name="text">
        <string>solid</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QRadioButton" name="radioBtnLineDash">
       <property name="text">
        <string>dashed</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>300</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_vertex">
     <item row="0" column="0">
      <widget class="QRadioButton" name="radioBtnVertexCircle">
       <property name="text">
        <string>circle</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QRadioButton" name="radioBtnVertexNo">
       <property name="text">
        <string>no</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QRadioButton" name="radioBtnVertexSquare">
       <property name="text">
        <string>square</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>690</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_move" columnstretch="0,0,0,0" rowminimumheight="0,0" columnminimumwidth="0,0,0,0">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="leftMargin">
      <number>1</number>
     </property>
     <property name="topMargin">
      <number>1</number>
     </property>
     <property name="rightMargin">
      <number>1</number>
     </property>
     <property name="bottomMargin">
      <number>1</number>
     </property>
     <item row="1" column="3">
      <widget class="QDoubleSpinBox" name="doubleSBMoveZ">
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="minimum">
        <double>-100.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="doubleSBMoveX">
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="minimum">
        <double>-100.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QDoubleSpinBox" name="doubleSBMoveY">
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="minimum">
        <double>-100.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="labelMoveX">
       <property name="text">
        <string>X</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="labelMoveZ">
       <property name="text">
        <string>Z</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="labelMoveY">
       <property name="text">
        <string>Y</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btnMove">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>710</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
color: black;
	background-color: rgb(255, 255, 171);
border-radius: 15px;
  border: 1px solid black;
}

QPushButton:pressed {
border: 2px solid black;
            background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #CA6C00, stop: 1 #F58C14);
}

</string>
    </property>
    <property name="text">
     <string>MOVE</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>690</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_rotate">
     <property name="leftMargin">
      <number>1</number>
     </property>
     <property name="topMargin">
      <number>1</number>
     </property>
     <property name="rightMargin">
      <number>1</number>
     </property>
     <property name="bottomMargin">
      <number>1</number>
     </property>
     <item row="1" column="0">
      <widget class="QDoubleSpinBox" name="doubleSBRotateX">
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="minimum">
        <double>-10.000000000000000</double>
       </property>
       <property name="maximum">
        <double>10.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="stepType">
        <enum>QAbstractSpinBox::DefaultStepType</enum>
       </property>
       <property name="value">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="doubleSBRotateY">
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="minimum">
        <double>-10.000000000000000</double>
       </property>
       <property name="maximum">
        <double>10.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QDoubleSpinBox" name="doubleSBRotateZ">
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="minimum">
        <double>-10.000000000000000</double>
       </property>
       <property name="maximum">
        <double>10.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="labelRotateX">
       <property name="text">
        <string>X</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="labelRotateY">
       <property name="text">
        <string>Y</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="labelRotateZ">
       <property name="text">
        <string>Z</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1093</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTolls">
    <property name="title">
     <string>☰</string>
    </property>
    <widget class="QMenu" name="menuSave_img">
     <property name="title">
      <string>Save img</string>
     </property>
     <addaction name="actionJpeg"/>
     <addaction name="actionBmp"/>
    </widget>
    <addaction name="actionSol"/>
    <addaction name="separator"/>
    <addaction name="actionSaveSettings"/>
    <addaction name="actionLoadSettings"/>
    <addaction name="separator"/>
    <addaction name="menuSave_img"/>
    <addaction name="actionGif"/>
    <addaction name="actionInfo"/>
   </widget>
   <addaction name="menuTolls"/>
  </widget>
  <action name="actionInfo">
   <property name="text">
    <string>Info</string>
   </property>
  </action>
  <action name="actionSol">
   <property name="text">
    <string>Projection</string>
   </property>
  </action>
  <action name="actionSaveSettings">
   <property name="text">
    <string>Save settings</string>
   </property>
  </action>
  <action name="actionLoadSettings">
   <property name="text">
    <string>Load settings</string>
   </property>
  </action>
  <action name="actionJpeg">
   <property name="text">
    <string>jpeg</string>
   </property>
  </action>
  <action name="actionBmp">
   <property name="text">
    <string>bmp</string>
   </property>
  </action>
  <action name="actionGif">
   <property name="text">
    <string>Gif</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Scene</class>
   <extends>QWidget</extends>
   <header>scene.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
